code/game/objects/structures/crates_lockers/closets/cardboardbox.dm

```diff
@@ -62,3 +64,17 @@
 			for(var/mob/M in viewers(src))
 				M.show_message("<span class='notice'>\The [src] has been cut apart by [user] with \the [WC].</span>", 3, "You hear cutting.", 2)
 			qdel(src)
+			return
+		if(istype(W, /obj/item/weapon/storage/fancy/crayons)) /////addendum by flimflamm
+			var/decalselection = input("Please select a decal") in list( "cancel", "atmospherics", "bartender", "barber", "blueshield",	"brig_physician", "captain",
+				"cargo", "chief_engineer",	"chaplain",	"chef", "chemist", "civilian", "clown", "cmo", "coroner", "detective", "engineering", "genetics", "hop",
+				"hos", "hydroponics", "internal_affairs_agent", "janitor",	"magistrate", "mechanic", "medical", "mime", "mining", "nt_representative", "paramedic", "pod_pilot",
+				"prisoner",	"research_director", "security", "syndicate", "therapist", "virology", "warden", "xenobiology")
+			if(decalselection == "cancel")
+				return
```

Comment:
@Krausus I think i successfully implemented adjacency check, item in hands check, incapacitation check, and also made it consume the crayon box upon completion. The only other potential thing that is left to add would be an associative list, but I already spruced up the sprite names to where they are quite legible as is.